relayer/workers/parachaincommitmentrelayer/parachain-commitment-listener.go

```diff
@@ -15,22 +15,25 @@ import (
 
 	"github.com/snowfork/polkadot-ethereum/relayer/chain"
 	"github.com/snowfork/polkadot-ethereum/relayer/chain/parachain"
+	"github.com/snowfork/polkadot-ethereum/relayer/chain/relaychain"
 	chainTypes "github.com/snowfork/polkadot-ethereum/relayer/substrate"
 )
 
 type ParachainCommitmentListener struct {
-	config   *parachain.Config
-	conn     *parachain.Connection
-	messages chan<- []chain.Message
-	log      *logrus.Entry
+	config               *parachain.Config
+	parachainConnection  *parachain.Connection
+	relaychainConnection *relaychain.Connection
```

Comment:
`relaychainConnection` is unused by this worker. Why was it added? Is it just scaffolding for future work?

Reply:
yep, sorry should have done in different PR, gonna leave it in for now as i'll be working on follow up PR today